geopandas/base.py

```diff
@@ -294,10 +332,9 @@ def sindex(self):
             self._generate_sindex()
         return self._sindex
 
-    def buffer(self, distance, resolution=16):
-        return gpd.GeoSeries([geom.buffer(distance, resolution)
-                             for geom in self.geometry],
-                         index=self.index, crs=self.crs)
+    def buffer(self, distance, **kwargs):
```

Comment:
Can we maybe explicitly pass the kwargs instead of `**kwargs` ? I find that more explicit for users